src/index.ts

```diff
@@ -191,11 +191,20 @@ function parseAddress(address: string): string {
     return address;
 }
 
-function parsePassphrase(passphrase: string): string {
+async function parsePassphrase(passphrase: string): Promise<string> {
     if (_.isUndefined(passphrase)) {
-        throw new CLIError(CLIErrorType.OptionRequired, {
-            optionName: "passphrase"
-        });
+        const Enquirer = require("enquirer");
+        const enquirer = new Enquirer();
+        enquirer.register("password", require("prompt-password"));
+        var questions = {
+            type: "password",
+            message: "Enter your passphrase please",
+            name: "passphrase"
+        };
+        await enquirer.ask(questions)
+            .then(function(answers: { passphrase: string }) {
+                passphrase = answers.passphrase;
+            });
```

Comment:
I think that using `await` and `.then` together seems strange.
This will be better.
```typescript
const askResult = await enquirer.ask(questions);
passphrase = askResult.passphrase;
```

Reply:
Let's just return the passphrase value.
```typescript
const askResult = await enquirer.ask(questions);
return askResult.passphrase;
```

Reply:
It looks better. I will fix it